exercises/connect/example.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * "Player O" plays from top to bottom, "Player X" plays from left to right.
+ * @param board
+ */
+export default class {
+  constructor(board) {
+    this.board = board.map(b => b.split(""));
+  }
+  winner() {
+    var players = ['X','O'];
+    for(let i = 0; i < 2; i++) {
```

Comment:
Let's use [for...of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) in place of an index, since we're only using the index to get each element.